lab_1/main.py

```diff
@@ -116,4 +303,11 @@ def save_profile(profile: dict) -> int:
     :param profile: a dictionary
     :return: 0 if everything is ok, 1 if not
     """
-    pass
+
+    if not isinstance(profile, dict):
+        return 1
+
+    file = open('../lab_1/profiles/' + profile['name'] + '.json', 'w')
```

Comment:
It is ok for now but I **highly** suggest reading about `os.path` and the `Path` library. The example could be found in the `start.py` file.